### PR TITLE
{lib}[GCCcore/6.4.0] ACTC v1.1

### PR DESCRIPTION
**Pull request by jdonners**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb]: @jdonners please include a `SHA256` checksum (you can use `eb --inject-checksums` for that)

File: easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+buildopts = 'CC="$CC" CFLAGS="$CFLAGS"'
```
Comment (boegel) [inline easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb]: please move this down, below `builddependencies` (above `files_to_copy` makes sense taking into account the order of steps)

File: easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+builddependencies = [
```
Comment (boegel) [inline easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb]: @jdonners At the risk of sounding annoying, `(build)dependencies` is usually included below the `sources` block...

Comment (jdonners) [reply easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb]: so you just wanted the `buildopts` to be moved down below builddependencies? ok..

Comment (boegel) [reply easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb]: Exactly.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/11ad83daecbf39383465555872ecac1c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/88d7e2bd5a365d509b4b7577f15d699f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jdonners!

**Pull request merged**